composer.json

```diff
@@ -1,26 +1,47 @@
 {
-  "name": "athens/encryption",
+  "name": "spryker/propel-encryption-behavior",
   "description": "Propel Behavior for seamless encryption/decryption of data columns",
-  "autoload": {
-    "classmap": [
-      "src/"
-    ]
-  },
-  "autoload-dev": {
-    "classmap": [
-      "test/"
-    ]
-  },
+  "type": "library",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Spryker Systems GmbH",
+      "homepage": "https://spryker.com"
+    }
+  ],
   "require": {
     "php": ">=7.1",
-    "propel/propel": "~2.0@dev"
+    "athens/encryption": "dev-master"
```

Comment:
Please revert this, as the dependency goes towards the lib, and no other outdated lib :)